pkg/cluster/operation/destroy.go

```diff
@@ -91,6 +90,46 @@ func Destroy(
 	return nil
 }
 
+// StopAndDestroyInstance stop and destroy the instance,
+// if this instance is the host's last one, and the host has monitor deployed,
+// we need to destroy the monitor, either
+func StopAndDestroyInstance(getter ExecutorGetter, compName string, cluster spec.Topology, instance spec.Instance, options Options, destroyMonitor bool) error {
```

Comment:
We can fetch the compName from `instance.ComponentName()`, so it can be removed at this time....

Reply:
Thanks for the advice.